barman/annotations.py

```diff
@@ -201,7 +201,7 @@ def get_annotation(self, backup_id, key, use_cache=True):
                 and (backup_id, key) not in self.annotation_cache
             ):
                 return None
-        # We either know there's an annotation or we havn't used the cache so read
+        # We either know there's an annotation or we haven't used the cache so read
```

Comment:
I noticed this when I first looked at the repository, but Google Sheets didn't suggest fixing it, and somehow I forgot that I really wanted to fix it...